server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/step/LoadReportAnalysisMetadataHolderStep.java

```diff
@@ -204,12 +208,11 @@ private static boolean reportBelongsToDefaultOrganization(@Nullable String organ
     return organizationKey == null || organizationKey.isEmpty();
   }
 
-  private Organization toOrganization(String organizationUuid) {
-    try (DbSession dbSession = dbClient.openSession(false)) {
-      Optional<OrganizationDto> organizationDto = dbClient.organizationDao().selectByUuid(dbSession, organizationUuid);
-      checkState(organizationDto.isPresent(), "Organization with uuid '%s' can't be found", organizationUuid);
-      return Organization.from(organizationDto.get(), organizationFlags.isEnabled(dbSession));
-    }
+  private Organization toOrganization(DbSession dbSession, String organizationUuid) {
+    analysisMetadata.setOrganizationsEnabled(organizationFlags.isEnabled(dbSession));
```

Comment:
`analysisMetadata.setOrganizationsEnabled` is already called on line 117 and it really does not belong to a method called `toOrganization` which really should be static